components/profile/follow/UserRow.vue

```diff
@@ -1,17 +1,11 @@
 <template>
   <div class="flex items-center justify-between">
-    <NuxtLink
-      class="flex"
-      :to="`/${urlPrefix}/u/${getss58AddressByPrefix(user.address, urlPrefix)}`">
-      <NuxtImg
-        :src="user.image"
-        placholder
-        alt="follower avatar"
-        class="w-12 h-12 rounded-full border object-cover mr-4" />
+    <NuxtLink class="flex" :to="`/${urlPrefix}/u/${prefixUserAddress}`">
+      <ProfileAvatar class="mr-4" :address="user.address" :size="48" />
```

Comment:
`!mr-4` since we haven't fixed the tailwindcss / bulma conflict issue

Reply:
updated

Reply:
> > since we haven't fixed the tailwindcss / bulma conflict issue
> 
> First time hearing that
> 
> is there any opened issue regarding to this?

https://github.com/kodadot/nft-gallery/issues/9973